docs/build-on-linea/gas-fees.mdx

```diff
@@ -1,139 +1,108 @@
 ---
-title: Gas fees on Linea
+description: How to estimate Linea gas costs
 sidebar_position: 6
 ---
 
-If you have interacted with the Ethereum ecosystem, then you've likely encountered the unpredictability of gas fees, which can swing widely based on network activity. 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
-For a refresher on gas, click [here](https://support.metamask.io/hc/en-us/articles/4404600179227-User-Guide-Gas#:~:text=A%20normal%20transaction%20sending%20ETH,transactions%20also%20cost%2021%2C000%20gas.). 
+# Estimate transaction costs
 
-As a Layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees. 
+Linea supports the [Ethereum EIP-1559 gas price model](https://ethereum.org/developers/docs/gas); however, being a
+layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees.
 
-**On average, gas fees on Linea are 15x cheaper than those on Ethereum.** Furthermore, we're continuously working towards reducing these costs even more to enhance affordability for our users.
+There are minor differences in the way Linea handles gas calculations to Ethereum:
 
-Linea is compatible with [EIP-1559](https://consensys.io/blog/what-is-eip-1559-how-will-it-change-ethereum); however, there are minor differences.
+- The base fee uses a set price of 7 wei. Blocks created by Linea use up to 30 million gas
+    (less than 50% of the maximum block size of 61 million gas), the fee decreases by 12.5% per block, effectively
+    keeping it at a stable 7 wei.
+- Transactions won't be sequenced if the `gasPrice` or `maxPriorityFeePerGas` falls below a certain threshold, which adjusts
+    over time. Instead, transactions are added to the pending queue until the gas price on the network drops sufficiently for the
```

Comment:
```suggestion
    over time and depend per transaction. Instead, transactions are added to the pending queue until the gas price on the network drops sufficiently for the
```

Reply:
Updated to "Transactions won't be sequenced if the `gasPrice` or `maxPriorityFeePerGas` falls below a certain threshold. This threshold is not static; it adjusts over time and varies depending on the specifics of each transaction.""